.github/workflows/depsreview.yml

```diff
@@ -0,0 +1,11 @@
+name: 'Dependency Review'
+on: [pull_request]
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@dcd71f646680f2efd8db4afa5ad64fdcba30e748
+      - name: 'Dependency Review'
+        uses: actions/dependency-review-action@3f943b86c9a289f4e632c632695e2e0898d9d67d
```

Comment:
@naveensrinivasan im wondering if there is some additional config required - the CI job always passes almost immediately - so im wondering if its doing anything

Reply:
AFAIK nothing. It checks its API for any vulnerabilities. So it should be quick.

Reply:
hmm - ok

what we do need to do is to limit this to the envoy repo only (ie not mirrors/forks etc)

see here for example of what i mean:

https://github.com/envoyproxy/envoy/blob/48cc862e6cb2497c7077925a9c21013c9f586a31/.github/workflows/pr_notifier.yml#L6-L10

is that something you would be able to follow up with @naveensrinivasan  ?

Reply:
actually we probably also want to filtuer on repo name or somesuch